src/components/UserList/UserListHead.tsx

```diff
@@ -22,18 +22,18 @@ const headCells: readonly HeadCell[] = [
   },
   {
     disablePadding: false,
-    id: 'room',
+    id: 'roomNumber',
     label: t('userList.roomNumber'),
   },
   {
     disablePadding: true,
-    id: 'userType',
+    id: 'roles',
     label: t('userList.userType'),
   },
 ];
```

Comment:
This should not be hard-coded. You should build a headers array from rows you are getting from the backend